From: Jiri Bohac <jbohac@suse.cz>
References: bsc#1219471
Subject: include 59-zfcp-compat.rules in kdump initrd

kdump uses a random one of the by-path symlinks to refer to the target
partition.

With 59-zfcp-compat.rules added to the SUSE package, symlinks in the form
/dev/disk/by-path/ccw-*.*.*-zfcp-*:*-part* are created. If kdump uses this symlink when generating
the kdump initrd it will fail on boot beacause the udev rule is missing in the kdump initrd
and the symlink not created in the kdump environment.

Fix this by including 59-zfcp-compat.rules in the kdump initrd.

--- a/zdev/dracut/95zdev-kdump/module-setup.sh	2024-02-21 15:57:33.027658387 +0100
+++ b/zdev/dracut/95zdev-kdump/module-setup.sh	2024-02-21 15:57:38.215675799 +0100
@@ -78,6 +78,7 @@
     inst_multiple /lib/s390-tools/zdev-from-dasd_mod.dasd
 
     inst_rules "59-dasd.rules"
+    inst_rules "59-zfcp-compat.rules"
 
     # Obtain kdump target device configuration

